pkg/attestation/crafter/api/attestation/v1/crafting_state.proto

```diff
@@ -150,6 +150,17 @@ message Attestation {
   // Signing options
   SigningOptions signing_options = 15;
 
+  // Workflow file path that was used during build 
+  string workflow_file_path = 16;
+
+  string runner_environment = 17;
```

Comment:
what does runner_environment mean?

Reply:
This is what is returned in the OIDC token, for example `github-hosted`. This is what GitHub SLSA attestation action is also reporting.

Reply:
can you add a comment then? Thanks!

Reply:
Sure thing, done.